core/src/test/scala/unit/kafka/tools/ConsumerPerformanceTest.scala

```diff
@@ -30,7 +30,7 @@ class ConsumerPerformanceTest {
   @Test
   def testHeaderMatchBody(): Unit = {
     Console.withOut(outContent) {
-      ConsumerPerformance.printHeader(true)
+      ConsumerPerformance.printHeader(true, false)
```

Comment:
Please provide parameter names. Also, let's add additional test cases so that we are covering all of the variations: detailed=true/false, consumer=old/new.